azkaban-executor/azkaban-emr-jobtype/src/main/java/uk/gov/dwp/dataworks/azkaban/app/App.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.dwp.dataworks.azkaban.app;
+
+import azkaban.utils.Props;
+import org.apache.log4j.Logger;
+import uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob;
+
+import static uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob.AWS_LOG_GROUP_PARAMETER_NAME;
+import static uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob.CLUSTER_PARAMETER_NAME;
+import static uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob.EMR_LAUNCHER_LAMBDA_PARAMETER_NAME;
+import static uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob.EXPORT_DATE_PARAMETER_NAME;
+import static uk.gov.dwp.dataworks.azkaban.jobtype.EmrLauncherJob.JOB_DEPENDENCIES_PARAMETER_NAME;
+
+public class App {
+    public static void main(String[] args) throws Exception {
+
+        Props jobProperties = Props
+                .of(JOB_DEPENDENCIES_PARAMETER_NAME, "ADG-incremental", EXPORT_DATE_PARAMETER_NAME, "2021-05-12",
```

Comment:
Why are these values are in code and not read in at runtime, we don't want to recompile the executors when we are writing new jobs. Is this just for testing?

Reply:
This app is purely for testing outside of azkaban, the parameters would ordinarily be job-specific parameters set in the job.properties file.